include/sea_dsa/Graph.hh

```diff
@@ -679,7 +680,9 @@ public:
 
   const Set getAccessedType(unsigned o) const {
     Offset offset(*this, o);
-    return m_accessedTypes.at(offset.getNumericOffset());
+    auto it = m_accessedTypes.find(offset.getNumericOffset());
```

Comment:
this seems unnecessary. There should be an assert in `at()` already

Reply:
there's no at in llvm::DenseMap

Reply:
there is `lookup`